packages/x-charts/src/internals/material/augmentation.ts

```diff
@@ -0,0 +1,37 @@
+import type { ButtonProps } from '@mui/material/Button';
+import type { IconButtonProps } from '@mui/material/IconButton';
+import type { DividerProps } from '@mui/material/Divider';
+import type { MenuItemProps } from '@mui/material/MenuItem';
+import type { MenuListProps } from '@mui/material/MenuList';
+import type { PopperProps } from '@mui/material/Popper';
+import type { TooltipProps } from '@mui/material/Tooltip';
+
+declare module '@mui/x-charts' {
```

Comment:
```suggestion
declare module '@mui/x-charts/models/slots/propOverrides' {
```

IIUC, you are trying to extend a type in this own module, `@mui/charts` so I think you need to use path-to-file in the declare.